v2/README.md

```diff
@@ -0,0 +1,223 @@
+# go-licenses
```

Comment:
Looks like this needs to be updated to reflect the current state.

Reply:
I have some more changes for readme in my fork's master branch.

I intend to do a batch update later because it will be some throw away work only to sync the readme with state of the PR, if we do it now.

What do you think?

Reply:
Let's try to keep this as up to the date to the current code as possible. Feel free to keep the current version in a fork and import back in later! :D

Reply:
That makes sense!
I updated README to only include information related to current state.